src/lib/journey/callbacks/kba/kba-create.svelte

```diff
@@ -18,12 +20,13 @@
   const prompt = callback.getPrompt();
   const questions = callback
     .getPredefinedQuestions()
-    ?.map((label, idx) => ({ text: label, value: idx }));
+    ?.map((label, idx) => ({ text: label, value: `${idx}` }));
```

Comment:
can `predefinedQuestions` be null or undefined? should we then do an optional chain on the`getPrompt` call? I feel like if we succeed the `prompt` callback call, then we would have predefined questions?

Reply:
`callback` will exist within this section of code, but there's not a guarantee of any predefined questions. There's a possibility of this callback only having custom questions. This is where the user defines their own question _and_ answer, rather than being given an existing question to answer.

It's worth noting that at this stage of development, none of the callback handler components are actually "done". The current story I'm working on will further develop the callback handlers associated with the default login tree. Then, there's another story that does the same for the default registration.

The current callback handler components you see here are developed just enough for testing the happy path.